workos/mfa.py

```diff
@@ -106,8 +106,9 @@ def get_factor(
             raise ValueError("Incomplete arguments. Need to specify a factor ID")
 
         response = self.request_helper.request(
-            "auth/factors/{authentication_factor_id}".format(
```

Comment:
What's the status of [formatted string literals](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings) in Python and our SDK Python version support? [Looks like it was added in 3.6](https://peps.python.org/pep-0498/) and our CI tests only go back to 3.8.

If we were to write `f"auth/factors/{authentication_factor_id}"` is there still a need for this method? Though I have one idea why we might want it (will leave a separate comment).